src/PROPOSAL/PROPOSAL/crosssection/parametrization/Photoproduction.h

```diff
@@ -34,9 +36,7 @@ namespace PROPOSAL {
                 return 1.; // all energy is always lost, i.e. v=1
             };
 
-            double GetLowerEnergyLim(const ParticleDef&, const Medium&, cut_ptr) const override {
-                return 2. * ME; // TODO: other limit?
```

Comment:
Does someone know, where the 2 m_e derives from?

Reply:
From the cutoff that we used for photons, because this is the energy below which no new electron-positron pairs can be produced.